srtcore/fec.cpp

```diff
@@ -453,7 +453,9 @@ void FECFilterBuiltin::feedSource(CPacket& packet)
     HLOGC(pflog.Debug, log << "FEC:feedSource: %" << packet.getSeqNo() << " rowoff=" << baseoff
             << " column=" << vert_gx << " .base=%" << vert_base << " coloff=" << vert_off);
 
-    if (vert_off >= 0 && sizeCol() > 1)
+    // [[assert sizeCol() >= 2]]; // see the condition above.
+
+    if (vert_off >= 0)
```

Comment:
Why don't you use `SRT_ASSERT` here?

```suggestion
    SRT_ASSERT(sizeCol() >= 2); // see the condition above.

    if (vert_off >= 0)
```

Reply:
The reason is explained in the comment.

Reply:
That looks like an explanation of an assertion, but not of the reason to convert it into a comment.

Reply:
This isn't a conversion. The assertion has been removed because it will never be reached. The comment is added to catch an eye in case someone is scared that this condition might be not checked.